Pspcl/Pspcl.Web/Views/Account/Login.cshtml

```diff
@@ -1,90 +1,58 @@
 ﻿
 @{
-    Layout = null;
+    Layout = "~/Views/Shared/NoSideBarLayout.cshtml";
 }
 
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Login</title>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
-    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
-    <link rel="stylesheet" href="~/Pspcl.Web.styles.css" asp-append-version="true" />
-</head>
-<body>
+@{
+    ViewData["Title"] = "Login";
+}
+
+@section styles {
     <style>
-        #electricMeter{
+        #electricMeter {
             height: 400px;
-            width: 400px;            
+            width: 400px;
         }
-        .bg{
-            background-color:#e3e4e5;
+
+        .bg {
```

Comment:
Avoid adding styles inline. It should be in a css file

Reply:
ok👍. understood. will keep in mind.